src/clj/ataru/tarjonta_service/kouta/kouta_client.clj

```diff
@@ -80,7 +81,7 @@
     :name                                                        (:nimi hakukohde)
     :tarjoaja-name                                               (or (:name (first tarjoajat)) {})
     :tarjoaja-oids                                               (mapv :oid tarjoajat)
-    :ryhmaliitokset                                              []
+    :ryhmaliitokset                                              hakukohderyhmas
```

Comment:
Kuinkahan tämä on aiemmin toiminut? Missä nuo aiemmin on populoitu? Mietin että eihän se nykyinen toteutus jyrää näitä sitten.

Reply:
Ymmärtääkseni vasta uuden hakukohderyhmäpalvelun myötä koutasta tuleville hakukohteille ollaan hakemassa niihin liittyvät hakukohderyhmät. Eli käsittääkseni nykyisellään jos hakukohde on koutasta tuleva, sillä ei ole ollut liittyviä hakukohderyhmiä.